### ISSUE TITLE
aiohttp spits SSL errors when dealing with invalide certificates

### ISSUE THREAD

**Issue by Unitiser** [delvelabs/hammertime#45]
I was running tachyon on a staging site that uses a self signed certificate when I got a bunch of SSL errors.

Here's a snippet of the error messages : 
```
SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f4bd11941d0>
transport: <_SelectorSocketTransport fd=51 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2605)
SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f4bd10b9470>
transport: <_SelectorSocketTransport fd=49 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2605)
SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f4bd10c5080>
transport: <_SelectorSocketTransport fd=45 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2605)
```

I believe this issue is linked to https://github.com/aio-libs/aiohttp/issues/3477. 

Updating aiohttp to 3.5.4 seams to fix the issue.

**Comment by lphuberdeau** [comment]
This is in fact a long standing issue I've seen once in a while. Can you confirm the aiohttp release fixes the issue?

**Comment by Unitiser** [comment]
After running tachyon on the same site with the updated version of aiohttp in my virtualenv, I did not get any errors. I did not make extensive tests tho.

**Comment by lphuberdeau** [comment]
Versions updated. Released as 0.6.2.

Disregard the failed build, that's travis attempting to publish the same version multiple times due to the build matrix.
